### PR TITLE
changed the innerHTML with textContent for security reasons

### LINKED ISSUE #10097: Update team page JavaScript to prevent XSS
### Problem

The content of our team page is largely generated with JavaScript.  When new elements are constructed for this page, text content is often added by setting the [`innerHTML` property](https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML).  This way of creating elements could leave us vulnerable to cross-site scripting attacks (more information [here](https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#security_considerations)).
 

### Reproducing the bug

1. Go to ...
2. Do ...

* Expected behavior:
* Actual behavior:

### Context

- Browser (Chrome, Safari, Firefox, etc): Any
- OS (Windows, Mac, etc): All
- Logged in (Y/N): N
- Environment (prod, dev, local): prod

### Breakdown

Since none of the text content used to generate the team member cards contains markup, it may be easiest to use `textContent` instead of `innerHTML` when creating the cards.

#### Requirements Checklist
* [ ] In `team.js`, replace all instances of `innerHTML` with `textContent`

#### Related files

* [`team.js`](https://github.com/internetarchive/openlibrary/blob/6224dce456619cbdbe740e40994b707b31f80019/openlibrary/plugins/openlibrary/js/team.js)

#### Stakeholders

*
<hr>

#### Instructions for Contributors

- Please [run these commands](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#working-on-your-branch) to ensure your repository is up to date **before** [creating a new branch](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#making-changes-and-creating-a-pull-request) to work on this issue and **each time after** pushing code to Github, because the pre-commit bot may add commits to your PRs upstream.

### PR DESCRIPTION
**Pull request by ronibhakta1**
Closes #10097 
Hey @jimchamp i have made the changes in innerHTML with textContent for security reasons as you mentioned. 

### Technical

changes in the `team.js` file to increase the security.

### Testing

no build errors and no other syntactical errors. all changes are preferable as per the documentations

### CODE REVIEW

**Review by jimchamp — CHANGES_REQUESTED** [review]
Thanks @ronibhakta1!  I've confirmed that everything is working.

Happy to merge this once the commented code on lines 134-137 are removed.

File: openlibrary/plugins/openlibrary/js/team.js [hunk 1]
```diff
@@ -130,13 +130,13 @@ export function initTeamFilter() {
...
                 : (memberName.classList = 'description__name--length-short');
 
-            memberName.innerHTML = `${member.name}`;
+            memberName.textContent = `${member.name}`;
             // memberRole.classList = 'description__role';
-            // memberRole.innerHTML = `${role}`;
+            // memberRole.textContent = `${role}`;
             // memberDepartment.classList = 'description__department';
-            // memberDepartment.innerHTML = `${member.departments}`;
+            // memberDepartment.textContent = `${member.departments}`;
```
[the remark this answers is not in the document]
Comment (ronibhakta1) [reply openlibrary/plugins/openlibrary/js/team.js:137]: Yeah sure i will update you with the changes

[the remark this answers is not in the document]
Comment (ronibhakta1) [reply openlibrary/plugins/openlibrary/js/team.js]: > Can you remove these comments altogether?

hey @jimchamp  i made the required changes you can check that out !

**Review by jimchamp — APPROVED** [review]
Thanks @ronibhakta1!

**Pull request merged**